utils/cache.py

```
@@ -43,36 +53,43 @@
-metrics = DozensMetrics()
+metrics = Metrics()
 
 
 class CachedTemplateView(TemplateView):
+    """A TemplateView that caches its rendered output.
+
+    This class method wraps the standard Django TemplateView with caching, improving performance for repeated requests.
+    """
+
     @classmethod
     def as_view(cls, **initkwargs):  # @NoSelf
```

Ruff (0.11.9)

Missing return type annotation for classmethod `as_view`

(ANN206)